src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -3707,10 +3707,10 @@ public String getTitle() {
     /**
      * dataset description
      *
-     * @return title of workingVersion
+     * @return description of workingVersion
      */
     public String getDescription() {
-        assert (null != workingVersion);
+        // TODO: Switch to `getDescriptionPlainText()` when we are ready to QA this.
```

Comment:
@pameyer wrote this code this code for #1393 and said he would prefer plain text. I don't think we want to re-test all that Zotero stuff right now though.